appeals/api/src/database/seed/data-static.js

```diff
@@ -390,6 +427,13 @@ export async function seedStaticData(databaseConnector) {
 			update: {}
 		});
 	}
+	for (const rejectionReason of representationRejectionReasons) {
+		await databaseConnector.representationRejectionReason.upsert({
+			create: rejectionReason,
+			where: { name: rejectionReason.name },
+			update: {}
+		});
+	}
```

Comment:
```suggestion
	const promises = representationRejectionReasons.map((rejectionReason) =>
		databaseConnector.representationRejectionReason.upsert({
			create: rejectionReason,
			where: { name: rejectionReason.name },
			update: {}
		})
	);

	await Promise.all(promises);
```

I want to suggest this but when I looked more I realised this is the last loop of a massive stack of suboptimal loops. We could _massively_ speed up this seed script. Take it or leave it, I'm happy to loop back later and fix all of these